client/src/plus/index.scss

```
@@ -487,7 +497,7 @@
           font-size: clamp(16px, calc(2vw * 0.6), 17px);
 
           @media #{$mq-large-desktop-and-up} {
-            font-size: clamp(16px, calc(2vw * 0.6), 18px);
+            font-size: clamp(16px, calc(2vw * 0.6), 17px);
```

Little scary for accessibility to set font sizes in pixels. Could we do:

```suggestion
            font-size: clamp($base-font-size, calc(2vw * 0.6), 1.1rem);
```